mozsci/ems.py

```diff
@@ -56,9 +60,9 @@ def select_ensemble_bagged(self, y, ymodels, verbose=False):
         indices = np.arange(len(ymodels))
         max_keep = int(self.pbags * len(ymodels))
         nmodels = 0
-        for k in xrange(self.nbags):
+        for k in range(self.nbags):
             if verbose:
-                print("Bagging number %s" % str(k+1))
+                print(("Bagging number %s" % str(k+1)))
```

Comment:
You shouldn't need to double parenthesize this. That's only necessary if you don't explicitly import `print_function`.